README.md

```diff
@@ -44,7 +47,7 @@ for other systems just look at maven central's dependency information.
 </dependency>
 ```
 
-#### Scala SBT
+#### Scala' sbt
```

Comment:
The quote shouldn't be there or there's a _s_ missing afterwards, I think.

Reply:
The `s` is omitted if the next letter is an `s` or sound like an `s`, but I'm not so sure about acronyms, I'll just remove it to be sure.

Reply:
Hmmm, no, the next word is not considered for the omission of the _s_ after the apostrophe.